### PR TITLE
Update docs and test cases to work with latest format versions

### PR DESCRIPTION
**Pull request by deitch**
**- What I did**
Updated the docs and tests to reflect some of the new (and renamed) formats per merged `moby` PR https://github.com/moby/tool/pull/169

Relevant format changes from that PR:

* new format `raw-efi` creates a minimal eft-bootable raw disk image
* new format `raw-bios` creates a minimal bios-bootable raw disk image
* renamed format `raw` to `aws` (since it was sized up to multiple of 1GB solely for AWS requirements)
* renamed format `qcow2` to `qcow2-bios` (since we will support qcow2 eft-bootable going forward

**- How I did it**
Updated the names of tests and `-format` references (where relevant) in docs.

**- How to verify it**

* docs: read them
* tests: CI should run and pass

**- Description for the changelog**

Updated tests and docs for latest moby formats

**- A picture of a cute animal (not mandatory but encouraged)**
ewok

### CODE REVIEW

**Comment by deitch** [comment]
Oops, forgot to sign. Repushed.

**Comment by justincormack** [comment]
You didnt update the `moby` commit in the `Makefile`

**Comment by deitch** [comment]
> You didnt update the moby commit in the Makefile

Drat, knew I must have missed something.

**Comment by deitch** [comment]
OK, `MOBY_COMMIT` updated. Letting CI run

**Comment by justincormack** [comment]
Hmm, mostly failed...

**Comment by deitch** [comment]
> Hmm, mostly failed...

That many tests changed, not too surprising. I will dig in.

**Comment by deitch** [comment]
> mostly failed.

All of the new tests. How do I get the output logs?

**Comment by deitch** [comment]
Let me rephrase. I see the output logs, but not for the tests themselves. It doesn't give any detail _why_ an individual test failed.

**Comment by justincormack** [comment]
There is a URL at the top of the test logs that goes to the detailed test logs, in this case https://github.com/linuxkit/linuxkit-logs/commit/12351d074e53a912cfc5004297d62f0be16048e1

**Comment by deitch** [comment]
Updated with new moby PR https://github.com/moby/tool/pull/175 ; do not approve or merge until Makefile updated with appropriate hash.

**Comment by justincormack** [comment]
Thats merged now

**Comment by deitch** [comment]
Great thanks. Will update the `MOBY_COMMIT` hash here, rerun tests, and then hopefully all clean.

**Comment by deitch** [comment]
Improved, still a test or two to fix.

**Comment by deitch** [comment]
Updated. One test left.

**Comment by deitch** [comment]
Ah, finally all green! @justincormack you approved before. Still good to go?

**Comment by deitch** [comment]
Thanks!
